Tests/Fluent.Tests/Cdn/CdnTests.cs

```diff
@@ -17,7 +17,7 @@ public partial class Cdn
         private const string cdnEndpointName = "endpoint-f3757d2a3e10";
         private const string endpointOriginHostname = "mylinuxapp.azurewebsites.net";
 
-        [Fact]
+        [Fact(Skip = "Fails with server side issue, need to follow up with the CDN service team")]
```

Comment:
Please log an issue for tracking, but not comment.

Reply:
It's copy-paste from sample cdn test here. I will try to fix later.